### DISCUSSION TITLE
How to make an equiangular Y?

### DISCUSSION THREAD

**Discussion by jason-s** [KaTeX/KaTeX#2555] [General]
I want to make an equiangular Y (for use as a subscript, not that this matters...) -- any suggestions?

The [Comprehensive LaTeX Symbol List](http://tug.ctan.org/info/symbols/comprehensive/symbols-a4.pdf#page=33) shows a couple of options like `\Ydown`:

![image](https://user-images.githubusercontent.com/738893/94589474-265e3900-023a-11eb-810d-5fddfd44f9db.png)

But these are outside the realm of normal KaTeX content.

If not, I suppose I can use `{\sf Y}` which sort of vaguely looks like the same shape.

![image](https://user-images.githubusercontent.com/738893/94591535-109e4300-023d-11eb-85a3-083f9e8bdcbe.png)

**Comment by k4b7** [comment]
These commands aren't currently supported by KaTeX, but you can open feature request ticket if you'd like.
